lib/ontologies_linked_data/models/mod/hydra_page.rb

```diff
@@ -0,0 +1,59 @@
+module LinkedData
+  module Models
+    class HydraPage < Goo::Base::Page
+
+      def convert_hydra_page(options, &block)
+        {
+          '@id': get_request_path(options),
+          '@type': 'hydra:Collection',
+          totalItems: self.aggregate,
+          itemsPerPage: self.size,
+          view: generate_hydra_page_view(options, self.page_number, self.total_pages),
+          collection: map { |item| item.to_flex_hash(options, &block) }
+        }
+      end
+      
+      def self.generate_hydra_context
+        {
+          'hydra': 'http://www.w3.org/ns/hydra/core#',
+          'Collection': 'hydra:Collection',
+          'member': 'hydra:member',
```

Comment:
here you call it `member`, but  use `collection` in the returned items.
Replace `collection` in line 12 by  `member`